### DISCUSSION TITLE
Can`t find nvim in Windows,but  nvim-data.how to config it

### DISCUSSION THREAD

**Discussion by Sizolity** [ayamir/nvimdots#1074] [Q&A]
do find nvim-data in Windows as the same way.i try to config it but failed

**Comment by ayamir** [comment]
`C:\Users\YOUR_USERNAME\AppData\Local\nvim`

**Reply by Sizolity** [reply]
errrrrrr ,didn't mean to disturb your time, follow this path to find only nvim-data.
 I just installed nvim without installing any plugins or anything else.
it take  a little bit time to learn how to manually install lazy.nvim and nvim

**Reply by charliie-dev** [reply]
> it take a little bit time to learn how to manually install lazy.nvim and nvim

that's why we include installation scripts, plz check https://github.com/ayamir/nvimdots/blob/main/scripts/install.ps1

**Reply by Sizolity** [reply]
haven`t installed terminal and can i run directly script on nvim
 I mean to download it directly on nvim   (A_A)
![IBEKJ 9({1XA}$Y)3U1MXMB](https://github.com/ayamir/nvimdots/assets/103719398/8cb229a0-7d1b-42d6-93bf-3d010c599db8)

**Reply by charliie-dev** [reply]
i have absolutely no idea about what your saying. 
since u look like a novice in the field, i would suggest u to look up some word definition and it's basic usage first, i.e. `terminal`, `gui/tui`, `XDG_CONFIG_HOME`, `%APPDATA%`, `%LOCALAPPDATA%`, `powershell script`, `bash script`.

And plz do read the repo [README](https://github.com/ayamir/nvimdots/blob/main/README.md) and [Wikis](https://github.com/ayamir/nvimdots/wiki).

**Comment by ayamir** [comment]
You can say Chinese directly if you are not good at English :)

**Reply by Sizolity** [reply]
感谢大佬们的包容。
因为已经安装了0.9.4 版的nvim，想着就抄个配置, 直接抄init.lua运行不了然后就按照教程走,然后就没有然后了
有没有能直接手动下载插件的方式,然后手动配置这个init.lua或者说有其他简单方法
恳请指条明路

**Reply by charliie-dev** [reply]
if u want to learn how to config nvim, i recommand this repo: https://github.com/nvim-lua/kickstart.nvim
it's structure is much simpler than ours, u can and should start from there.
